src/components/organisms/about/about.scss

```
@@ -0,0 +1,34 @@
+@use "../../variables" as *;
+
+.container {
+  padding: 0px;
+  margin: 100px 0px;
+  @media screen and (max-width: $breakpoint-lg) {
+    margin: 40px 0px;
+  }
```

```suggestion
  @media screen and (max-width: $breakpoint-lg) {
    padding-top: 40px;
    padding-bottom: 40px;
  }
```
こちらも上下にのみpaddingをつけたいので、こんな感じに修正お願いします！